frontend/src/metabase/admin/settings/components/widgets/FormattingWidget.tsx

```diff
@@ -158,7 +158,7 @@ export function FormattingWidget() {
                 { label: "100 000,00", value: ", " },
                 { label: "100.000,00", value: ",." },
                 { label: "100000.00", value: "." },
-                { label: "100'000.00", value: ".'" },
+                { label: "100’000.00", value: ".’" },
```

Comment:
we should also have a frontend test for this so that the materiality of this is documented